### PR TITLE
merge commit master to ergon-master

### PR DESCRIPTION
**Pull request by shanselm-ergon**
merge the current remote master into ergon-master

### CODE REVIEW

File: ios/Plugin/HttpRequestHandler.swift [hunk 1]
```diff
@@ -190,10 +190,9 @@ class HttpRequestHandler {
         }
 
         let urlRequest = request.getUrlRequest()
-        
         let session = URLSession(configuration: URLSessionConfiguration.default, delegate: allowAllDelegate, delegateQueue: nil)
-        
         let task = session.dataTask(with: urlRequest) { (data, response, error) in
+            session.invalidateAndCancel()
```
Comment (7-plus-t) [inline ios/Plugin/HttpRequestHandler.swift:195]: Why is this needed? And if it is needed, wouldn't `finishTasksAndInvalidate()` be the right thing to do?

Comment (7-plus-t) [reply ios/Plugin/HttpRequestHandler.swift:195]: Change happened here (without any reasoning): https://github.com/capacitor-community/http/pull/157/commits/eaaedecd8921c358447fa66314b36b1151da586c

Comment (shanselm-ergon) [reply ios/Plugin/HttpRequestHandler.swift:195]: Yes, I also saw this and merged it (this seams to be an improvement in the HTTP plugin). I couldn't figure out, why this is needed, but i removed the semicolon at the end.

**Review by k-graefenhain — APPROVED** [review]

File: ios/Plugin/HttpRequestHandler.swift [hunk 2]
```diff
@@ -190,10 +190,9 @@ class HttpRequestHandler {
         }
 
         let urlRequest = request.getUrlRequest()
-        
         let session = URLSession(configuration: URLSessionConfiguration.default, delegate: allowAllDelegate, delegateQueue: nil)
-        
         let task = session.dataTask(with: urlRequest) { (data, response, error) in
+            session.finishTasksAndInvalidate()
```
Comment (shanselm-ergon) [inline ios/Plugin/HttpRequestHandler.swift:195]: @7-plus-t i changed the method, I also think this fits better.
